MPlotTest.pro

```diff
@@ -4,8 +4,12 @@
 #
 #-------------------------------------------------
 
+greaterThan(QT_MAJOR_VERSION, 4): QT += widgets printsupport
+
 TEMPLATE = app
 TARGET = MPlotTest
+
+CONFIG += depend_includepath
```

Comment:
To be consistent with your other changes this should be wrapped in a `greaterThan()` line.

Reply:
Requested macro is added.